### ISSUE TITLE
max trading pairs per group

### ISSUE THREAD

**Issue by en3z** [taniman/profit-trailer-enhancements#859]
If we can set max_trading_pairs per group, we will be able to take advantage of the new group feature in version 2.5.

ex)
default_max_trading_pairs = 3 (no group)
GROUP1_max_trading_pairs = 2
GROUP2_max_trading_pairs = 2

**Comment by taniman** [comment]
i do not see the logic behind this request. Please explain what it is you are trying to achieve with this

**Comment by en3z** [comment]
Set up group 1 and group 2 coins and give them different strategies. And I want to manage risk by setting max pairs for each group.

**Comment by taniman** [comment]
I have 8 pairs currently. The bot does not know that  3 belong to group1 and 5 belong to group 5

And if the bot knew this. 
Are you saying that you will set group 1 max trading pairs = 8 
group2 max trading pairs is 5

So total trading pairs is 13?

If that is what you are saying, I understand the request, otherwise I do not

**Comment by en3z** [comment]
correct. Please understand that English is not my native language and my explanation is lacking.

**Comment by taniman** [comment]
default_max_trading_pairs = 3 (no group)
GROUP1_max_trading_pairs = 2
GROUP2_max_trading_pairs = 2

going further on this, does it mean that we can have 8 pairs trading?
And does max_trading_pairs  count GROUP max trading pairs as one of it's trading pairs too?
I would say  yes.


Can you give me a scenario where this is usefull? Why would setting a  global max trading pairs not be enough?

**Comment by en3z** [comment]
I want to try.  Group 1 as a long-term strategy and Group 2 as a short-term strategy.  And I want to limit the risk of each long-term and short-term.  Global max pairs only limit the overall risk, so if there are many short-term (group 2) positions, can long-term (group 1) not be able to enter new positions at all?  Now that only kyc-certified accounts on Binance use the api, I can't do much account diversification. I can use other exchanges, but I want to reduce fragmentation.
To be honest, this request is probably not a necessary feature.  It's just that I want to test the possibilities.  So, if you don't feel the need, you don't have to. Because PT still has a very high degree of freedom and I am satisfied.
